### ISSUE TITLE
User story - Job location type

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#33]
### If applicable, add a related Epic.

Epic: #28 

### User Story

As a **recruiter user**, I can **specify the job location type** so that I can **ensure only those who match the job location, such as on-site, hybrid or remote show interest in the job**.

### Acceptance Criteria

- The option to select remote, hybrid or on-site is provided
- The option is reflected on the frontend

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Set up the JobLocationType table
- [x] Ensure intermediary tables are set up for JobPostLocationType and TechUserLocationType
- [x] Use a select or radio input in the form

**Comment by stephendawsondev** [comment]
After reviewing the user stories, I have moved this to a `should have` as another user story needed to be added, which is a must have.

**Comment by stephendawsondev** [comment]
Moving this to Sprint 3 - it's almost done, but I need to add the location preferences for the tech user signup and edit, as well as a way to display that on their profiles. While the job post uses a radio button, I'm considering having checkboxes for the user preferences.

**Comment by stephendawsondev** [comment]
This was completed with #89
